simplepay.php

```diff
@@ -3,7 +3,7 @@
 Plugin Name: SimplePay
 Plugin URI: http://www.simplepay.ng
 Description: Online and Mobile Payment. Secure. Simple.
-Version: 1.5.5
+Version: 1.5.6
 Author: SimplePay (support@simplepay.ng)
 Author URI: http://www.simplepay.ng
 *

```

Comment:
Please update 'SP_PLUGIN_VERSION' => '1.5.5' to 'SP_PLUGIN_VERSION' => '1.5.6' on line 35